core/src/main/scala/org/apache/spark/ui/PagedTable.scala

```diff
@@ -154,9 +150,6 @@ private[ui] trait PagedTable[T] {
    * }}}
    */
   private[ui] def pageNavigation(page: Int, pageSize: Int, totalPages: Int): Seq[Node] = {
-    if (totalPages == 1) {
-      Nil
-    } else {
```

Comment:
One more comment: need to adjust the indent of the following code block.

Reply:
Thanks for reviewing, 
handled your comment, showing more difference because of white-space.
please check